specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2024-05-01-preview/azureopenai.json

```diff
@@ -0,0 +1,2845 @@
+                },
+                "Create a user-compute ingestion job": {
+                    "$ref": "./examples/put_ingestion_job_uc.json"
+                }
+            }
+        },
+        "get": {
+            "tags": [
+                "Ingestion:"
+            ],
+            "summary": "Gets the details of the specified job id.",
+            "operationId": "IngestionJobs_Get",
+            "produces": [
+                "application/json"
+            ],
+            "parameters": [
+                {
+                    "$ref": "#/parameters/apiVersionQueryParameter"
+                },
+                {
+                    "in": "path",
+                    "name": "job-id",
+                    "required": true,
+                    "description": "The id of the job.",
+                    "type": "string"
+                }
+            ],
+            "responses": {
+                "200": {
+                    "description": "Success",
+                    "schema": {
+                        "$ref": "#/definitions/IngestionJob"
+                    }
+                },
+                "default": {
+                    "description": "An error occurred.",
+                    "schema": {
+                        "$ref": "#/definitions/ErrorResponse"
+                    }
+                }
+            },
+            "x-ms-examples": {
+                "Get ingestion job": {
+                    "$ref": "./examples/get_ingestion_job.json"
+                }
+            }
+        }
+    },
+    "/ingestion/jobs/{job-id}/runs": {
+        "get": {
+            "tags": [
+                "Ingestion:"
+            ],
+            "summary": "Lists the runs of the specified job id.",
+            "operationId": "IngestionJobRuns_List",
+            "produces": [
+                "application/json"
+            ],
+            "parameters": [
+                {
+                    "$ref": "#/parameters/apiVersionQueryParameter"
+                },
+                {
+                    "in": "path",
+                    "name": "job-id",
+                    "required": true,
+                    "description": "The id of the job.",
+                    "type": "string"
+                },
+                {
+                    "in": "header",
+                    "name": "mgmt-user-token",
+                    "type": "string",
+                    "description": "The token used to access the workspace (needed only for user compute jobs)."
+                }
+            ],
+            "responses": {
+                "200": {
+                    "description": "Success",
+                    "schema": {
+                        "$ref": "#/definitions/IngestionJobRunList"
+                    }
+                },
+                "default": {
+                    "description": "An error occurred.",
+                    "schema": {
+                        "$ref": "#/definitions/ErrorResponse"
```

Comment:
Is this really using the same error contract as OAI?

Reply:
@johanste For the ingestion APIs, we are using the same error contract as the other authoring APIs e.g. finetuning, ...etc. That's for the API-level errors. For the run-level failures, we introduced a new contract that is specific to OYD ingestion upon your and Jeffrey's suggestion (since it's returned in the payload of a successful GET request)